config/prod/s3-cfn-templates.yaml

```diff
@@ -0,0 +1,9 @@
+template_path: "remote/S3/public-bucket.yaml"
+stack_name: "cloudformation-templates"
+stack_tags:
+  Department: "CompOnc"
+  Project: "imCORE"
+  OwnerEmail: "bruno.grande@sagebase.org"
+hooks:
+  before_launch:
+    - !cmd "curl {{stack_group_config.aws_infra_templates_root_url}}/S3/public-bucket.yaml --create-dirs -o templates/remote/S3/public-bucket.yaml"
```

Comment:
please use `wget` instead of `curl`.  it's better because it has retries built in. here's an example, https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/sceptre/scicomp/config/prod/rotate-credentials.yaml#L12